### ISSUE TITLE
Merge to alternative branch.

### ISSUE THREAD

**Issue by KendallDoesCoding** [refined-github/refined-github#6585]
### Description

Add a option, to merge PR to a alternate branch or tag, for example:

- It will be used to merge to branch to test and then merge to main
- It can be merged ot a BETA tag and go into the beta release. 

### Example URLs

https://github.com/KendallDoesCoding/mogul-christmas

**Comment by fregante** [comment]
> * It can be merged ot a BETA tag

You can't merge into a tag, tags are constant (unless you want to delete it and create a new one), which is basically like a force push and not something Refined GitHub can/should do.



> to merge PR to a alternate branch

You can already do that by changing the base branch of the PR.

If your intention is to merge into another branch _but_ keep the PR open, then it's been requested in:

- https://github.com/refined-github/refined-github/issues/5938

**Comment by fregante** [comment]
Duplicate of https://github.com/refined-github/refined-github/issues/5938
